samples/README.md

```diff
@@ -46,7 +46,7 @@ View the [source code](https://github.com/googleapis/nodejs-game-servers/blob/ma
 __Usage:__
 
 
-`node samples/create_cluster.js`
+`node samples/create_cluster.js <project_id> <location> <realmId> <clusterId> <gke_cluster_name>`
```

Comment:
Just to make sure - this file is auto-generated by `synthtool`.  Did you run `synthtool`, or hand edit?

Reply:
Thanks. Hand edited. Should I revert my change. The README.md is clearly missing information.

Reply:
No harm in having it here, the robot account will just overwrite any inconsistencies when it runs after this PR lands.